charts/cluster-proportional-autoscaler/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: cluster-proportional-autoscaler
+version: 1.0.0
```

Comment:
Trying to understand - so this is the version being presented in the helm chart? In the future if we need to update the image version, or adjust some configurations in the template file, we will bump this version, is that correct?

Reply:
It sounds like your understanding of the concepts is correct but I will provide additional detail just to help clarify.

Version is the version of the chart itself and will be increased to release new versions of the chart.  Things like changes to the RBAC or adding a network policy, etc.  So, depending on how you want to handle chart releases you will either want chart PRs to increase this number if you want them to release as soon as they are merged to master or if you prefer to delay the release then you would want the PR to not change this version and you would set it on master when ready to release.  As new versions are released they will be listed in the chart repo, which the index for is what will be in the gh-pages branch, allowing people to choose the version of the chart they would like to use.

appVersion is the default version of the proportional autoscaler image that will be used unless image.tag is provided as a value to helm install/upgrade.  This can be seen [here](https://github.com/kubernetes-sigs/cluster-proportional-autoscaler/pull/113/files#diff-b0620dbcddd12f184e07f3e718d25b54793a3420babab54c38087d2d812bce87R36)

Reply:
Thank you for the thorough explanation. That is super helpful.